### DISCUSSION TITLE
Using Curator outside a Filament Panel

### DISCUSSION THREAD

**Discussion by ijpatricio** [awcodes/filament-curator#345] [Q&A]
Hey @awcodes , everyon!

How's it going?

Is it possible to use Curator in a Form outside Filament Panel?

Didn't find it in docs.

In the picture: regular Filament's `FileUpload` filed, and below, a `CuratorPicker`.
It renders well, and when clicked it does trigger an action - second photo.

In this page, which is not a Panel, I'm rendering a Filament Form, and having `<x-filament-actions::modals />` for a delete confirmation, for example.

What I don't know is what component to include so that where `CuratorPicker` can mount on.

Any pointers??

Thanks!

![image](https://github.com/awcodes/filament-curator/assets/26031459/a9c0ae70-0bdc-4265-a4e2-d0f95f3510a1)
![image](https://github.com/awcodes/filament-curator/assets/26031459/79821a0b-551a-49f3-9549-22c1ec79dffc)

**Comment by awcodes** [comment]
Hmm. It's just a mounted form component action. As long as the modals are there it should be fine.

**Reply by ijpatricio** [reply]
It isn't. Maybe I'm doing something wrong...

`<x-filament-actions::modals />` is there, and it's working for the form component when deleting an item from a `Repeater`.

Is the "modals" component you refer, the same one?

<img width="1137" alt="image" src="https://github.com/awcodes/filament-curator/assets/26031459/3590bd84-31ed-4887-9f1d-841f6529c2ce">

**Reply by awcodes** [reply]
I'll try to recreate as soon as I get a chance.

**Reply by ijpatricio** [reply]
I'll have an example installed readily for you to fork and edit, on 1-click, with Gitpod

**Reply by awcodes** [reply]
sure. send me the link
